sdk/keyvault/azure-security-keyvault-keys/src/test/java/com/azure/security/keyvault/keys/KeyAsyncClientTest.java

```diff
@@ -80,6 +80,16 @@ public void createRsaKey(HttpClient httpClient, KeyServiceVersion serviceVersion
             .verifyComplete());
     }
 
+    /**
+     * Tests that a Ec key created.
+     */
+    @ParameterizedTest(name = DISPLAY_NAME_WITH_ARGUMENTS)
+    @MethodSource("getTestParameters")
+    public void createEcKey(HttpClient httpClient, KeyServiceVersion keyServiceVersion) {
+        createKeyAsyncClient(httpClient, keyServiceVersion);
+        createEcKeyRunner((expected) -> client.createEcKey(expected));
```

Comment:
It is important that we assert that the result of the operation is correct, so I would at least add one assertion like on the method above (`createRsaKey`).